db/people.yml

```diff
@@ -561,15 +561,12 @@ stao:
       department: apam
       group: bg
       location: New York, NY
-      other:
```

Comment:
You should add this back.  This is not an empty line, it contains a list with one element.  In yaml when you have
```
this:
  - that
  - the other
```
it resolves in python as ``{"this": ["that","the other"]``